.env

```diff
@@ -0,0 +1,8 @@
+# This is an example environment file for configuring dynamic values.
```

Comment:
Is this an example or is this actually used? I.e., the comments says it's an "example", but this seems to be filling in  default values, which implies it's really used, in which case, it is not an example, really...

Reply:
Renamed it as `example.env` and removed sensitive information. Updated the readme as well.